app/components/QRCode/QRCodeModalBackground.tsx

```diff
@@ -1,9 +1,9 @@
 import { View, Image } from 'react-native'
 
-const background = require('../assets/gradient.png')
+const background = require('../../../assets/gradient.png')
 
 interface QRCodeModalBackgroundProps {
-  style: any
+  style: object
```

Comment:
Should be better to use `style?: StyleProp<ViewStyle>` instead of object ? (I think, i don't know types on RN)